### PR TITLE
allow object extensions

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -7,7 +9,10 @@ export interface TokenInfo {
   readonly logoURI?: string;
   readonly tags?: string[];
   readonly extensions?: {
-    readonly [key: string]: string | number | boolean | null;
+    readonly [key: string]:
+      | { [key: string]: ExtensionValue }
+      | { [key: string]: ExtensionValue }[]
```
Comment (moodysalem) [inline src/types.ts:14]: idt array is allowed, can you confirm?

**Review by macotouenaca — APPROVED** [review]

**Review by Hjordaan2002 — APPROVED** [review]

**Review by endless61 — APPROVED** [review]

File: src/types.ts (hunk 1 above)
Comment (Shahryarmohammadie643) [reply src/types.ts:14]: Yes

**Comment by Padraic-O-Mhuiris** [comment]
This is a great addition, am hoping it can be merged soon

**Comment by tinahaibodi** [comment]
Hello! Just wanted to see when this would be merged?

**Review by Shahryarmohammadie643 — APPROVED** [review]
Ok

**Review by moodysalem — APPROVED** [review]

**Review by Shahryarmohammadie643 — COMMENTED** [review]
Hi

File: src/types.ts (hunk 1 above)
Comment (Keyfo7272) [reply src/types.ts]: Ok

**Review by Kinou90100 — COMMENTED** [review]
#128 
#127 
#113
#105 
#81 
#73 
@Christoc90

**Review by Kinou90100 — COMMENTED** [review]
#128 
#127
#105 
#81 
#73
@Christoc90

**Review by Magkoooh — APPROVED** [review]

File: src/types.ts (hunk 1 above)
Comment (daglizzy2003) [reply src/types.ts]: Hello

**Pull request merged**